tests/foreman/cli/test_oscap.py

```diff
@@ -497,11 +542,20 @@ def test_postive_create_scap_policy_with_valid_name(self):
 
         :expectedresults: The policy is created successfully.
 
-        :caseautomation: notautomated
+        :caseautomation: automated
```

Comment:
we do not use that tag in case test is automated as far as I know. Applied to all other tests